mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -676,6 +676,7 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 		METHOD method_40143 isIn (Lnet/minecraft/class_6885;)Z
 			ARG 1 blocks
 		METHOD method_40144 streamTags ()Ljava/util/stream/Stream;
+		METHOD method_41520 getBlockRegistryEntry ()Lnet/minecraft/class_6880;
```

Comment:
The one in FluidState is just called `getRegistryEntry`. Given block states aren't registered it should be obvious this one returns the entry for blocks.